examples/rotation_bin/rotation_bin/mod.rs

```diff
@@ -0,0 +1,100 @@
+mod imp;
+
+use gtk::glib;
+use gtk::subclass::prelude::*;
+
+#[derive(Debug, Clone, Copy, Eq, PartialEq, glib::Enum)]
+#[repr(i32)]
+#[enum_type(name = "RotationEnum")]
```

Comment:
Why `RotationEnum` and not just `Rotation`?

Reply:
I assume that's the GType name so having it be just Rotation seems a bit weird. Would be fine if the types where prefixed I guess.

Reply:
Putting the kind into the type name seems very strange though :)

Reply:
Indeed, and I don't see any other prefix in the rest of the examples, so let's go without I guess.